blackbox/docs/configuration.txt

```diff
@@ -403,7 +403,7 @@ Ports
   of the node is not directly reachable from outside, e.g. running it
   behind a firewall or inside a Docker container.
 
-.. _psql.port:
+.. _psql_port:
```

Comment:
Are you sure this change is correct?
See failure: https://travis-ci.org/crate/crate/jobs/165487788#L1720